rho/scancommand.py

```diff
@@ -459,7 +459,10 @@ def _validate_options(self):
 
         # perform fact validation
         facts = self.options.facts
-        if facts == [] or facts == ['default'] or facts == ['all']:
+        if facts == []:
```

Comment:
Should not empty and default mean the same?

Reply:
I thought it would be confusing since the DEFAULT_FACTS object exists, which includes everything. The goal is to change this back in the next version when we "improve" jboss scanning. I'm flexible either way, we don't currently talk about 'default' in the docs. We mostly kept it to keep the project consistent as we changed things over time.

I'm open to preference on this one though.

Reply:
I don't have a preference here, just wanted to make sure that the default and blank are or are not required to mean the same.

I think we can merge this then.